### DISCUSSION TITLE
ODB: Is there any way to parsing two technology (lef file) in one odb database?

### DISCUSSION THREAD

**Discussion by ApeachM** [The-OpenROAD-Project/OpenROAD#3642] [Q&A] [answered]
I am writing some code for 3D Placer with ODB. 
Here, the technology of the two dies will differ. 

However, ODB is not currently supporting the 3D placer data structure.

If I want to read two technologies, then should I make different two dbDatabase objects for parsing two different lef file?
Or is there another way to parse two lef files just using one dbDatabase object?

Thanks for the time for reading my question.

Sincerely.

**Comment by maliberty** [comment, marked as answer]
There is no way currently to support two lef files.  That's an enhancement we would make as part of the Darpa 3d-ic program if we are funded.

**Reply by ApeachM** [reply]
Thanks for your reply!
